infra/src/dsl/value-files-generators/local-setup.ts

```diff
@@ -118,10 +146,13 @@ export const getLocalrunValueFile = async (
     .map((port) => `-p ${port}:${port}`)
     .join(
       ' ',
-    )} -v ${process.cwd()}/${defaultMountebankConfig}:/app/default.json bbyars/mountebank:2.8.1 start --configfile=/app/default.json`
+    )} -v ${process.cwd()}/${defaultMountebankConfig}:/app/default.json docker.io/bbyars/mountebank:2.8.1 start --configfile=/app/default.json`
 
   return {
-    services: { ...dockerComposeServices },
+    services: Object.entries(dockerComposeServices).reduce(
+      (acc, [name, service]) => ({ ...acc, [name]: service.command![0] }),
```

Comment:
should we be asserting `service.command![0]`

Reply:
Excellent point, fixed.